modules/book.js

```diff
@@ -0,0 +1,35 @@
+const books = { tab: [] };
```

Comment:
- Optional
Instead of declaring the `books` variable as a `const` and then having a `tab` array inside it so you can manipulate it, it would have been better if you declared `books` as `let`. That way you didn't have to make `books` an object and you would have been able to mutate the `books` directly.

Reply:
Hi @muneebulrehman, thank you for your suggestion, I would modify it before merging.
Can you also please give me some suggestions to solve the bundle errors that appear in the console